src/Resources/app/storefront/src/checkout/confirm-order.plugin.js

```diff
@@ -269,15 +272,15 @@ export default class ConfirmOrderPlugin extends Plugin {
 
         try {
             if ('isAvailable' in paymentMethodInstance) {
-                paymentMethodInstance.isAvailable().then(() => {
-                    $('#confirmOrderForm button[type=submit]').remove();
-                    let confirmButtonContainer = $('<div id="adyen-confirm-button" data-adyen-confirm-button></div>');
-                    $('#confirmOrderForm').append(confirmButtonContainer);
-                    paymentMethodInstance.mount(confirmButtonContainer.get(0));
-                }).catch(e => {
+                paymentMethodInstance.isAvailable().then(function () {
+                    this.mountPaymentButton(paymentMethodInstance);
+                }.bind(this)).catch(e => {
                     console.log(selectedPaymentMethodObject.type + ' is not available', e);
                 });
+            } else {
```

Comment:
`paymentMethodInstance` for Paypal does not have `isAvailable()` and can be mounted right away